### DISCUSSION TITLE
Failed to run workflow_dispatch

### DISCUSSION THREAD

**Discussion by azro352** [community/community#141694] [Mobile] [answered]
When trying to run a workflow from the mobile app, after adding the required values, the button to run keeps being disabled 
I trying changing the branch, and all the values, in every order 

![image](https://github.com/user-attachments/assets/aad7cbcc-1b5a-4fd2-8ef4-ec8c4863c1c8)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.182.0 (144168013)
Device: iPhone SE (2nd generation) (iOS 18.0.1)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @azro352!

Thank you for reporting this. I've investigated it, found the cause, and merged a fix which should become available in the 1.184.0 release. 

Thank you again!

**Reply by azro352** [reply]
1.183.1 fixed it !
Thanks
